public/js/jibe.js

```diff
@@ -339,22 +345,39 @@ var Jibe = (function (BCSocket, CodeMirror, Replay, showdown, Timestamps, TextFo
       replay.timestamps.colors = timestamps.colors;
 
       if ($('#replay-controls-container').is (':visible')) {
-        replay.stop ();
+        if(!replay.stopped){
+         replay.stop ();
+        }
         $('#play-button').removeClass('glyphicon glyphicon-stop').addClass('glyphicon glyphicon-play');
 
         $('#replay-controls-container').hide ("fast");
+
         $('#entry-markdown-replay').next ('.CodeMirror').hide ();
         $('#entry-markdown').next ('.CodeMirror').show ();
-        replay.reset ();
-        timestamps.draw (timestamps.lines);
+
         $('div#editor-preview-container').removeClass ('replaying');
         $('#editor-preview-toggle').bootstrapToggle ('enable');
+        $('#format-bold').prop("disabled",false);
```

Comment:
Consider using a css class to select all relevant buttons at once - `$('.format-button').prop('disabled', false');`

Also use single quotes.